src/main/resources/templates/index.html

```diff
@@ -8,7 +10,7 @@
 <script>
 function showError(message) {
   document.getElementById("error").style.visibility = "visible";
-  document.getElementById("error").innerHTML = message;  
+  document.getElementById("error").innerHTML = message;
```

Comment:
Apparently IntelliJ insisted that there shouldn't be whitespace at the end of these lines, so they're not really related to this PR ^^